packages/roosterjs-editor-types/lib/interface/EditorOptions.ts

```diff
@@ -145,6 +145,11 @@ export default interface EditorOptions {
      */
     imageSelectionBorderColor?: string;
 
+    /**
+     * Initial Table Border Format
+     */
+    initialTableBorder?: Border;
```

Comment:
what is this value for?

Reply:
The Initial Table Border Format passed to the editor.

Reply:
Why editor need to know the initial table border color?

Reply:
It is an optional parameter, same as imageSelectionBorderColor, to allow customization of what will be the initial format if you were to apply table borders.

Reply:
Oh, do you mean it is allowed that user format table border without specifying a format?

I think this should be part of the UI layer, but not the roosterjs editor layer.

For example, in roosterjs we should always require a valid set of parameters of the border when do border format. Then in UI layer (here it means OWA/Monarch) it can store the default settings somewhere, and when format table border, pass the stored default value to roosterjs. In that case, roosterjs doesn't need to know the concept of default border format at all.

Please refer to the text color and background color button in OWA, they have the similar behavior.

Reply:
Table borders should not be allowed to apply without a format. If the user does not specify one on editor creation, a default one will be set in createContentModelEditorCore.ts.

I do not think the current standard of undefined for Text color or background color are right, but even those are also at the editor lever through defaultFormat. Why would this one not be?

Reply:
I understand what you mean. So there are two different features.

About the text default format, it is the format for text even before user do any format change. Editor still need to format it as the default format, so editor needs to remember this format.

For table border, I agree there needs to be a default format, and actually there is one inside insertTable.ts. User can change to use a different format when insertTable is called. But since it is only used when user insert a table or format a table, there is entrypoint to pass the format in, not like the default text format. So that editor does not need to remember table border format.